cypress/e2e/yves/customer-account-management/customer-auth-dms.cy.ts

```diff
@@ -0,0 +1,67 @@
+import { container } from '@utils';
+import { LoginPage, CustomerOverviewPage } from '@pages/yves';
+import { CustomerAuthDynamicFixtures, CustomerAuthStaticFixtures } from '@interfaces/yves';
+import {
+  CreateStoreScenario,
+  UserLoginScenario,
+  EnableCmsBlockForAllStoresScenario,
+} from '../../../support/scenarios/backoffice';
+import { SelectStoreScenario } from '../../../support/scenarios/yves';
+import { faker } from '@faker-js/faker';
+
+(Cypress.env('isDynamicStoreEnabled') ? describe : describe.skip)('customer auth dms', { tags: '@dms' }, () => {
+  describe('customer auth', { tags: ['@customer-account-management'] }, (): void => {
+    const userLoginScenario = container.get(UserLoginScenario);
+    const createStoreScenario = container.get(CreateStoreScenario);
+    const selectStoreScenario = container.get(SelectStoreScenario);
+    const enableCmsBlockForAllStoresScenario = container.get(EnableCmsBlockForAllStoresScenario);
+    const loginPage = container.get(LoginPage);
+    const customerOverviewPage = container.get(CustomerOverviewPage);
+
+    let dynamicFixtures: CustomerAuthDynamicFixtures;
+    let staticFixtures: CustomerAuthStaticFixtures;
+
+    before((): void => {
+      ({ staticFixtures, dynamicFixtures } = Cypress.env());
+
+      userLoginScenario.execute({
+        username: staticFixtures.rootUser.username,
+        password: staticFixtures.defaultPassword,
+      });
+
+      createStoreScenario.execute({
+        store: staticFixtures.store,
+      });
+
+      enableCmsBlockForAllStoresScenario.execute({
+        cmsBlockName: 'customer-registration_token--text',
+        storeName: staticFixtures.store.name,
+      });
+      enableCmsBlockForAllStoresScenario.execute({
```

Comment:
why it's needed? You just need to check that login/registration is success.

Reply:
It's needed for enabling email template which is used when customer is registered